plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/SnykToolView.java

```diff
@@ -261,6 +281,75 @@ public void toggleIgnoresButtons() {
 
 	}
 
+	/*
+	 * Sets up for doing a Net New Issues scan.
+	 */
+	public void enableDelta() {
+		clearDeltaNodeListeners();
+
+		Display.getDefault().asyncExec(() -> {
+			if (this.treeViewer != null && !this.treeViewer.getTree().isDisposed()) {
+
+				TreeItem[] rootItems = getTreeViewer().getTree().getItems();
```

Comment:
why not use getRoot()?

Reply:
I need to TreeItems to update the text on them. Or is there another way to do this?